keyserver/handler.go

```diff
@@ -19,10 +19,22 @@ func (server *ConiksServer) handleRegistrationMessage(reg *p.RegistrationRequest
 			p.Error(p.ErrorMalformedClientMessage)
 	}
 
+	server.Lock()
+	// check the temporary bindings array first
+	// currently the server allows only one registration/key change per epoch
+	if server.tbs[reg.Username] != nil {
```

Comment:
If we already have a TB for the name, do we want to return some kind of proof to the client along with the error? cc @arlolra @Liamsi